app/src/view/screens/PostLetter/PostLetterInfo.js

```
@@ -227,65 +224,66 @@ export default function PostLetterInfo({ navigation }) {
   };
-  console.log('user-----------about ----', authToken )
-  const fetchCategory = async (token, lang)  => {
+  console.log("user-----------about ----", authToken);
+  const fetchCategory = async (token) => {
     // const token = token;
-    console.log('user-----------', authToken )
+    console.log("user-----------", authToken);
     try {
       const response = await fetch(
-        base_url + 'discCategory/getAllDiscCategories?page=1&limit=10000',
+        base_url + "discCategory/getAllDiscCategories?page=1&limit=10000",
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)